tasks/configure_namespace.yml

```diff
@@ -85,29 +85,39 @@
   - configmaps
 
 
-# TODO: When definitions are available in atomic-reactor
-- name: Create Pipeline from definition stored in repository
-  k8s:
-    definition: '{{ item }}'
-    namespace: "{{ osbs_ocp_namespace }}"
-  with_items: '{{ lookup("url", "{{ osbs_a_r_pipeline_definition }}", split_lines=False) | from_yaml_all | list }}'
-  when: item is not none
+- name: Download Pipeline / Tasks definitions
+  get_url:
+    url: "{{ item.url }}"
+    dest: "/tmp/{{ item.filename }}"
+    mode: '0640'
+  with_items: "{{ osbs_a_r_pipeline_tasks_definitions }}"
+  tags:
+  - task-definitions
+
+
+- name: Ensure OCP definitions have namespace define
+  lineinfile:
+    path: "/tmp/{{ item.filename }}"
+    insertafter: 'metadata:\n'
+    state: present
+    line: '  namespace: "{{ osbs_ocp_namespace }}"'
+  with_items: "{{ osbs_a_r_pipeline_tasks_definitions }}"
   tags:
-  - pipeline-task-definitions
+  - task-definitions
 
 
-# TODO: When definitions are available in atomic-reactor
-- name: Create Tasks from definition stored in repository
+- name: Create Pipeline / Tasks
   k8s:
-    definition: '{{ item }}'
-    namespace: "{{ osbs_ocp_namespace }}"
```

Comment:
Wait, so the `namespace` param doesn't work? It has to be in the resource definition?

Reply:
https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/ansible/roles/operator-pipeline/tasks/main.yml#L267

Look at how they have it done, something is very fishy here

Reply:
Problem here was not namespace but my ignorance when it comes to ansible. When I passed lines with definition from file via `with_items` I started wondering, OK and now how can I pass variable with multiple files with yaml definitions, when you can use with_items only once per task... but I figured out now. Preparing fix